handbook/openid-connect/openid-connect.md

```diff
@@ -338,3 +367,22 @@ To resolve this issue:
 3. Modify the number at the end to a number larger than X, where X is the required size of the cache entry specified in the error log.
 
 4. Restart the NI Web Server from the `Control` tab of the NI Web Server Configuration application.
+
+### ID Token Management Encryption
+
+If the provider is configured to use an asymmmetric ID token management encryption algorithm and private keys are missing or incorrect, or if the provider is configured to use an unsupported ID token management encryption algorithm, the user will be redirected back to the SystemLink login page after logging in to the provider.
+
+!!! note "Example error logs"
+    When this happens, the NI Web Server error logs will contain entries containing the following:
+
+    ```text
+    oidc_proto_parse_idtoken: oidc_jwt_parse failed
+    ```
+
+    The log entry will also contain more information about the problem.
+
+To resolve this issue:
+
+1. Confirm that the provider is configured to use supported encryption and signing algorithms. See [**Supported Signing and Encryption Algorithms**](#supported-signing-and-encryption-algorithms).
```

Comment:
Would 'uses' or 'is using' do the same work here? 

I get the feeling we want to drive home that this is all configurable. In that case, would saying 'In [the configuration utility or something], confirm the provider is using supported encryption...' work to give them more direct resolution guidance?

Reply:
I referred to the provider's documentation b/c the details of where to check the configuration is going to depend on the provider. Let me know what you think.